htdocs/core/class/fileupload.class.php

```diff
@@ -489,7 +489,7 @@ public function get()
 	public function post()
 	{
 		if (isset($_REQUEST['_method']) && $_REQUEST['_method'] === 'DELETE') {
-			return $this->delete();
+			echo $this->delete();
```

Comment:
If you remove the return here, it meas, once the action detete has been done, we output the result code on a page, but the code continue and we do the rest of code.
We must keep a return here to stop the action.